app/services/search_service.rb

```diff
@@ -90,7 +90,7 @@ def url_resource_symbol
   def full_text_searchable?
     return false unless Chewy.enabled?
 
-    statuses_search? && !@account.nil? && !((@query.start_with?('#') || @query.include?('@')) && !@query.include?(' '))
+    statuses_search? && !@account.nil? && !((@query.start_with?('#') || @query.match(/^(@\w+|@?\w+@[\w.]+)$/)) && !@query.include?(' '))
```

Comment:
I'd prefer to remove that check instead of having to maintain that regex.

Reply:
Completely remove regex and always do full text search?